runtime/gc_include/j9modron.h

```diff
@@ -98,6 +98,8 @@ typedef enum {
 	j9gc_modron_configuration_compressObjectReferences, /* a UDATA (TRUE or FALSE) representing whether or not object references are compressed */
 	j9gc_modron_configuration_heapRegionShift, /* a UDATA representing the shift amount to convert an object pointer to the address of the region */
 	j9gc_modron_configuration_heapRegionStateTable, /* a pointer to the base of the region state table */
+	j9gc_modron_configuration_gcConcurrentThreadCount,  /* a UDATA representing the MAX number of GC threads being used as Concurrent Mark Helpers */
```

Comment:
let's make the comment more generic: being used during concurrent GC operations

Reply:
Right. Missed update for this comment after we decided to extend request functionality. I am going to update this comment after completion of compilation jobs (in the case of you are not going to have functional change requests).